monadic-html/src/main/scala/mhtml/package.scala

```diff
@@ -1,12 +1,7 @@
-import scala.language.implicitConversions
 
 import scala.xml.Node
 
 package object mhtml {
-  implicit def RxToOSN[A](a: Rx[A]): Node                     = new xml.Atom(a)
```

Comment:
Sweet! Removing these implicit conversions is IMO enough motivation to move forward with this :smile: